### PR TITLE
Add profileCard.jsx

### PR DESCRIPTION
**Pull request by gauravsinhaweb**

### CODE REVIEW

**Review by Mohitk391 — APPROVED** [review]
Everything looks great, one thing you can improve afterward is spacing between the content body and navigation bar, other than that you can merge the commit.

**Comment by gauravsinhaweb** [comment]
> Everything looks great, one thing you can improve afterward is spacing between the content body and navigation bar, other than that you can merge the commit.

Thanks for your suggestion, I've fixed it.

**Review by sanket4120 — APPROVED** [review]
Good to merge

File: src/components/cards/FeedCard.jsx [hunk 1]
```diff
@@ -21,7 +22,17 @@ import {
 import { CommentsDisplay } from "../index";
```
Comment (HeyNitin) [inline src/components/cards/FeedCard.jsx:22]: you can have a look at absolute imports

**Review by HeyNitin — APPROVED** [review]
Left some Suggestion, rest looks good to merge

**Pull request merged**